docs/install/supported_operators_k8s.md

```diff
@@ -0,0 +1,28 @@
+# Supported operators and K8s clusters
+
+Percona Everest provides support for various operators and Kubernetes clusters. The following is a list of the specific operators and Kubernetes clusters that are compatible with Percona Everest:
+
+## Operators
+
+* Percona Operator for MySQL Based on Percona XtraDB Cluster (PXC) 1.14.0, 1.15.0
+* Percona Operator for MongoDB (PSMDB) 1.15.0
+* Percona Operator for PostgreSQL (PG) 2.3.1, 2.4.1
```

Comment:
Since we're here we might as well update these
```suggestion
* Percona Operator for MySQL Based on Percona XtraDB Cluster (PXC) 1.14.0, 1.15.0
* Percona Operator for MongoDB (PSMDB) 1.17.0, 1.18.0
* Percona Operator for PostgreSQL (PG) 2.4.1, 2.5.0
```